src/activities/categorization/CategoryReview.qml

```diff
@@ -76,7 +75,7 @@ Item {
             x: leftScreen.width + middleScreen.width + 0.01 * parent.width
             z: 2
             anchors.top: categoryBackground.top
-            anchors.topMargin: items.mode != "expert" ? rootItem.categoryImage.height + 0.027 * rightScreen.height : 0.05 * categoryBackground.height
+            anchors.topMargin: !(!items.instructionsVisible && !items.scoreVisible) ? rootItem.categoryImage.height + 0.027 * rightScreen.height : 0.05 * categoryBackground.height
```

Comment:
just for info (amit fixed it in another way), the simplification is "items.instructionVisible || items.scoreVisible" with a OR